packages-node/nodejs-helpers/test-node/index.test.js

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import {
+  parseCode,
+  transformCode,
+  prettify,
+  makeDirSync,
+  makeDir,
+  byStringAscendingSort,
+  camelToKebabCase,
+  getExportSpecifiersByFile,
+  asyncSerialForEach,
+  asyncConcurrentForEach,
+  // Tasks
+  bypassImportMap,
+  bypassExportMap,
+} from '../src/index.js';
+
+describe('Public API', () => {
+  it('should expose the agreed public API', () => {
+    expect(parseCode).to.exist;
```

Comment:
maybe we can test whether they are functions, since otherwise the import statement above would already have thrown an error?

Reply:
Done